flytekit/bin/entrypoint.py

```diff
@@ -262,6 +264,28 @@ def _execute_task(task_module, task_name, inputs, output_prefix, raw_output_data
                 _handle_annotated_task(task_def, inputs, output_prefix, raw_output_data_prefix)
 
 
+@_scopes.system_entry_point
+def _execute_map_task(task_module, task_name, inputs, output_prefix, raw_output_data_prefix, max_concurrency, test):
+    task_module = _importlib.import_module(task_module)
+    task_def = getattr(task_module, task_name)
+
+    if not test and isinstance(task_def, PythonFunctionTask):
+        map_task = MapPythonTask(task_def, max_concurrency)
+
+        task_index = _compute_array_job_index()
+        output_prefix = _os.path.join(output_prefix, str(task_index))
+
+        _handle_annotated_task(map_task, inputs, output_prefix, raw_output_data_prefix)
+
+    # Old style task
+    elif not test:
+        raise _system_exceptions.FlyteSystemAssertion(
+            "Map tasks are only supported for task of type `PythonTask` got task {} of type {} instead".format(
```

Comment:
can you also say, only supported for the typehinted sdk?

Reply:
done